etna/lib/etna/clients/magma/workflows/add_project_models_workflow.rb

```diff
@@ -21,7 +21,7 @@ def prepare_models_from_csv(csv_io, &err_block)
           models = csv_lines.inject(Models.new) do |acc, n|
             line_no += 1
             ModelsCsv.apply_csv_row(acc, n) do |err|
-              err_block.call("Error detected on line #{line_no}: #{err}")
+              err_block.call("Error detected on line #{line_no + 1}: #{err}")
```

Comment:
Due to the CSV headers adding an extra line, I think we should add `1` to the line number reported to the user. It's a bit more intuitive.